extensions/wikia/AdEngine/js/lookup/prebid/adapters/wikia.js

```diff
@@ -0,0 +1,120 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.wikia',[
+	'wikia.document',
+	'wikia.window',
+	'wikia.querystring'
+], function (doc, win, QueryString) {
+	'use strict';
+
+	var bidderName = 'wikia',
+		slots = {
+			oasis: {
+				TOP_LEADERBOARD: {
+					sizes: [
+						[728, 90]
+					]
+				},
+				TOP_RIGHT_BOXAD: {
+					sizes: [
+						[300, 250]
+					]
+				}
+			},
+			mercury: {
+				MOBILE_TOP_LEADERBOARD: {
+					sizes: [
+						[320, 50]
+					]
+				}
+			}
+		},
+		qs = new QueryString();
+
+	function getPrice() {
+		var price = qs.getVal('wikia_adapter', 0);
+
+		return parseInt(price, 10) / 100;
```

Comment:
Price in QueryString can be only decimal? Because it is always rounded to decimal, so 5.3 will not pass